fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -602,10 +585,16 @@ protected void replaceResourceWithStream(final FedoraResource resource,
     }
 
     protected void patchResourcewithSparql(final FedoraResource resource,
-                                           final String requestBody,
-                                           final RdfStream resourceTriples)
-            throws MalformedRdfException, AccessDeniedException {
-        resource.updateProperties(translator(), requestBody, resourceTriples);
+            final String requestBody,
+            final RdfStream resourceTriples)
+                    throws MalformedRdfException, AccessDeniedException {
+        if (resource instanceof NonRdfSourceDescription) {
+            // update the description instead
```

Comment:
+1 to public forum.
Yes, I misread @escowles comment: `My understanding was that the way we were going to handle these competing priorities was to keep fcr:metadata, make sure all binary properties used it as the RDF subject`.
My understanding was that the properties were to be stored on `fcr:metadata` but have the subject of the binary.